hubblestack/extmods/modules/hstatus.py

```diff
@@ -11,12 +16,50 @@ def get():
 
         (probably only useful from other excution modules)
     '''
-    return hubblestack.status.HubbleStatus.as_json()
+    return hubblestack.status.HubbleStatus.stats()
+
+def msg_counts(pat=r'hubblestack.hec.obj.input:(?P<stype>[^:]+)', reset=True):
```

Comment:
Lots of one-letter variable names in this one. It makes the code pretty hard to follow. `s` could be `stats`, `r` could be `ret`, `m` could be `match`, and `d` could be `matchdict` or something. It's a little more verbose, but brings some clarity.